subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/recomp/AbstractRecompilationSpecProvider.java

```diff
@@ -42,6 +42,10 @@
 import java.util.Set;
 
 abstract class AbstractRecompilationSpecProvider implements RecompilationSpecProvider {
+
+    private static final String MODULE_INFO_CLASS = "module-info";
+    private static final String PACKAGE_INFO_CLASS = "package-info";
```

Comment:
What do you think about _CLASS_NAME? I am thinking to use that, since we use these two for example when calling:
```
sourceFileClassNameConverter.getRelativeSourcePaths(MODULE_INFO_CLASS_NAME)
```
Where parameter of getRelativeSourcePaths is `className`:
```
Set<String> getRelativeSourcePaths(String className)
```